Sources/SPMTestSupport/MockDependencyGraph.swift

```diff
@@ -71,7 +71,7 @@ private extension MockPackageContainer {
                 .map { constraint in
                     switch constraint.requirement {
                     case .versionSet(let versionSet):
-                        return (constraint.identifier.identity, versionSet)
+                        return (container: constraint.identifier.identity.description, versionRequirement: versionSet)
```

Comment:
should we be passing the identity directly instead of it's description?

Reply:
See https://github.com/apple/swift-package-manager/pull/3032#discussion_r521326243